README.md

```diff
@@ -59,6 +74,27 @@ bedrock.start();
 
 For documentation on server configuration, see [config.js](./lib/config.js).
 
+### IP Tunnel
+Bedrock server maybe used with an IP tunnel.
+
+In `config/express.js`:
+```js
+// allow the tunnel http only access
+config.express.httpOnly = true;
+// configure fastify to trust the ip tunnel proxy 
+config.express.fastifyOptions.trustProxy = true;
+```
+
+In `config/server.js`:
+```js
+// accept connections from any interface on your device
+config.server.domain = '0.0.0.0';
```

Comment:
@mattcollier, it seems a little odd to me that this isn't using `bindAddr` here but `domain`:

```suggestion
config.server.domain = '0.0.0.0';
```